spec/features/login/log_as_admin_user_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+feature "Successful login" do
+  before do
+    @user = AdminUser.create!(email: "admin@example.com", password: "password", password_confirmation: "password")
+  end
+
+  background do
+    visit "/admin"
+  end
+
+  scenario "Check login page exists and login" do
+    expect(page).to have_content("Login")
+
+    expect(page).to have_css("div", text: "You need to sign in or sign up before continuing.")
+
+    within("form#session_new") do
+      expect(page).to have_content("Remember me")
+    end
+
+    expect(page).to have_current_path("/admin/login")
+
+    within("li#admin_user_email_input") do
```

Comment:
This within is too specific. You search field by ID so it should be only one. You can use within for entire form to speed up the test